### PR TITLE
Upgrade to 2020.1

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description

Upgraded FPS project to 2020.1.

#### Tests

- [x] Local launch
- [x] Cloud launch

### CODE REVIEW

**Comment by paulbalaji** [comment]
needs to target `develop`

File: workers/unity/Packages/manifest.json [hunk 1]
```diff
@@ -16,13 +17,15 @@
     "io.improbable.gdk.transformsynchronization": "0.3.10",
     "com.unity.modules.ai": "1.0.0",
     "com.unity.modules.animation": "1.0.0",
+    "com.unity.modules.assetbundle": "1.0.0",
```
Comment (paulbalaji) [inline workers/unity/Packages/manifest.json:20]: what is this?

Comment (jamiebrynes7) [reply workers/unity/Packages/manifest.json:20]: Part of the engine that's been moved into packages I think. If you don't have this, you get errors saying a type has been redirected and to install this.

**Review by paulbalaji — APPROVED** [review]

**Review by zeroZshadow — APPROVED** [review]

**Pull request merged**